iroh-net/src/net/network.rs

```diff
@@ -0,0 +1,46 @@
+use std::net::{IpAddr, Ipv4Addr, Ipv6Addr};
+
+/// Network selection between Ipv4 and Ipv6.
+#[derive(Debug, Copy, Clone, PartialEq, Eq)]
+pub enum Network {
```

Comment:
As this is now fully `pub`, can we **please, pretty please** rename this to `IpFamily`.  This name has been bothering me since the start...

Reply:
done